src/ServiceProviders/PaymentGateways.php

```diff
@@ -109,6 +108,9 @@ public function boot()
 
         $this->registerMigrations();
         $this->registerStripeCustomFields();
+
+        // Setup PayPal Commerce action/filter hooks.
+        (new PayPalCommerce())->boot();
```

Comment:
It might be better to separate the Gateway logic and the boot/hook logic. It would probably make sense to add a Service Provider for the PayPal gateways - I'm thinking of doing the same for Stripe.

Reply:
Why is booting necessary within the gateway? This feels like a flawed limitation within the API if it's necessary.

Reply:
@JasonTheAdams @kjohnson this is old code that I did not remove from class. I will remove this from class.

Reply:
@kjohnson I move action/filter hooks to service provider: https://github.com/impress-org/givewp/compare/a746a95d222a...3aa61ac2de8d